packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -19,7 +19,7 @@ const AccountMenuItems = props => {
     const menu = menuItems.map(item => {
         return (
             <Link className={classes.link} key={item.name} to={item.url}>
-                <FormattedMessage id={item.id} />
+                <FormattedMessage id={item.id} default={item.id} />
```

Comment:
As we're moving the ID's to not be English strings anymore we should update the default here to be the English version.

Is this meant to be `default` or `defaultMessage`?